internal/render/render.go

```diff
@@ -18,16 +20,40 @@ var (
 )
 
 func init() {
+	initEmptyTmpls()
+	for _, name := range bindata.AssetNames() {
+		tmpls = template.Must(tmpls.Parse(string(bindata.MustAsset(name))))
+	}
+}
+
+// LoadCustomTemplates allows to load in custom templates from a specified path.
+func LoadCustomTemplates(dir string) error {
+	initEmptyTmpls()
```

Comment:
I believe you should be able to remove this call to `initEmptyTmpls()`. Does it crash if you do? If it doesn't cause an error, please revert `init` to how it was before.